packages/blade/src/_helpers/storybook/Sandbox/Sandbox.web.tsx

```diff
@@ -23,48 +27,38 @@ const getBladeVersion = (): string => {
     return `https://pkg.csb.dev/razorpay/blade/commit/${shortSha}/@razorpay/blade`;
   }
 
-  return '*';
+  return 'https://pkg.csb.dev/razorpay/blade/commit/b1b0f883/@razorpay/blade';
 };
 
 const bladeVersion = getBladeVersion();
 
-function Sandbox({
```

Comment:
Explanation of changes:
- Moved the "setup" logic of Sandbox to `useSandbox` hook from `Sandbox` component
- Created `SandboxProvider` component that just does the setup without the layout of editor
- Same setup logic is used in `SandboxProvider`

If you want normal layout (like we have at the start of all stories, we use the same `<Sandbox />` component. Nothing changed there.

If you want custom layouts, we use `<SandboxProvider>` and use the `Sandpack*` layout components to place them as we like.

E.g. In responsive props docs, I am showing 2 previews. One for desktop and one for mobile
<img width="836" alt="Sandbox Provider usage from Box.stories" src="https://user-images.githubusercontent.com/30949385/222672152-87e60b50-c84f-47e9-8abe-af25ca0bf248.png">